src/Dockerfile

```diff
@@ -6,12 +6,17 @@ COPY Inkluzitron/*.csproj Inkluzitron/
 RUN dotnet restore Inkluzitron/ -r linux-x64
 COPY . .
 RUN dotnet publish Inkluzitron/ -c release -o /app -r linux-x64 --self-contained false --no-restore
+RUN mkdir /app/fonts
+RUN cp /source/Inkluzitron/Resources/Fonts/*.ttf /app/fonts/
 
 FROM mcr.microsoft.com/dotnet/runtime:5.0-buster-slim
-RUN apt-get update && apt-get -y install libgdiplus tzdata && apt-get clean
+RUN apt-get update && apt-get -y install libgdiplus tzdata fontconfig && apt-get clean
 ENV TZ=Europe/Prague
 WORKDIR /app
 COPY --from=build /app .
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+RUN mv /app/fonts/* /usr/share/fonts
```

Comment:
.NET nenatahuje fonty z /usr/share/fonts v základu? :peepocantbelievethisshit:

Reply:
Natahuje, akorát to nikdo nezkusil, fixed